CMakeLists.txt

```diff
@@ -217,8 +217,11 @@ set(EXTRA_FLAGS
     "-Werror=strict-aliasing"
     "-frounding-math"
     "-fsignaling-nans"
+    "-D_GLIBCXX_ASSERTIONS"
```

Comment:
I've added this flag to the default build to help prevent introducing code that might cause problems again.